HaruhiGekidouCLI/ArcCommand.cs

```diff
@@ -72,17 +72,22 @@ public override int Invoke(IEnumerable<string> arguments)
         else if (_pack)
         {
             GekidouArc arc = new();
-            string[] directories = Directory.GetDirectories(_input, "*", SearchOption.AllDirectories).Order().ToArray();
+            List<string> directories = [];
+            directories.Add(_input);
+            directories.AddRange(Directory.GetDirectories(_input, "*", SearchOption.AllDirectories).Order().ToArray());
```

Comment:
```suggestion
            string[] directories = 
            [
                _input,
                .. Directory.GetDirectories(_input, "*", SearchOption.AllDirectories).Order()
            ];
```
More efficient to initialize this as an array this way.